src/test/scala/viper/server/core/ViperServerHttpSpec.scala

```diff
@@ -113,5 +123,12 @@ class ViperServerHttpSpec extends AnyWordSpec with Matchers with ScalatestRouteT
         contentType should ===(ContentTypes.`application/json`)
       }
     }
+
+    "should eventually stop" in {
+      failAfter(Span(10, Seconds)) {
+        Await.ready(viperServerHttp.stopped(), Duration.Inf)
```

Comment:
Maybe the test should fail after some number of seconds?

Reply:
That's why I'm using `failAfter(Span(10, Seconds))`